src/treetensornetworks/projttns/projttnsum.jl

```diff
@@ -1,17 +1,25 @@
 """
 ProjTTNSum
 """
-struct ProjTTNSum{V}
-  pm::Vector{ProjTTN{V}}
-  function ProjTTNSum(pm::Vector{ProjTTN{V}}) where {V}
-    return new{V}(pm)
+struct ProjTTNSum{T<:AbstractProjTTN}
+  pm::Vector{T} where {T}
+  function ProjTTNSum(pm::Vector{T}) where {T}
+    return new{T}(pm)
   end
 end
+#ToDo: define accessor functions
 
 copy(P::ProjTTNSum) = ProjTTNSum(copy.(P.pm))
 
+# The following constructors don't generalize well, maybe require to pass AbstractProjTTN instead of TTN and remove these? 
```

Comment:
It seems reasonable to have a simple constructor that assumes it will be `ProjTTN`.